WebPlayer/TranscriptViewer/Quest 5.9 Online Transcript Viewer.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
I think we should rename this file to `index.html`, then the URL to it should be `https://play.textadventures.co.uk/TranscriptViewer` (I think!)

Reply:
> (I think!)

I'll test that real quick before submitting these changes.

Reply:
Yep. Works like a charm. (I was wondering if it needed a trailing `/`, but that is added automatically (of course).)

Good thinking!

All these changes have been committed.